File: src/main/carteira-investimentos/src/app/model/ativo.ts
export class Ativo {
    codigo: string;
    empresa: string;
    setor: string;
    precoMedioIR: number;
    precoMedioReal: number;
    quantidade: number;
    capitalInvestido: number;
    ultimoFechamento: number;
    capitalAtual: number;
    variacaoPercentual: number;
    variacaoFinanceira: number;

    constructor(codigo: string, empresa: string, setor: string, precoMedioIR: number, precoMedioReal: number,
         quantidade: number, capitalInvestido: number, ultimoFechamento: number, capitalAtual: number,
         variacaoPercentual: number, variacaoFinanceira: number) {
             this.codigo = codigo;
             this.empresa = empresa;
             this.setor = setor;
             this.precoMedioIR = precoMedioIR;
             this.precoMedioReal = precoMedioReal;
             this.quantidade = quantidade;
             this.capitalInvestido = capitalInvestido;
             this.ultimoFechamento = ultimoFechamento;
             this.capitalAtual = capitalAtual;
             this.variacaoPercentual = variacaoPercentual;
             this.variacaoFinanceira = variacaoFinanceira;
         }
}

File: src/main/java/br/com/investimentos/investimentos/model/Ativo.java
package br.com.investimentos.investimentos.model;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Entity
@Table(name="ativo")
@Data
public class Ativo {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Long id;
	private String codigo;
	private String empresa;
	private String setor;
	private double precoMedioIR;
	private double precoMedioReal;
	private int quantidade;
	private double capitalInvestido;
	private String tipo;

}
